marpa/lib/Guacamole.pm

```diff
@@ -194,7 +194,8 @@ ExprCondR     ::= ExprRange0  OpTriThen ExprRangeR OpTriElse ExprCondR | ExprRan
 ExprAssignL   ::= ExprCond0   OpAssign  ExprAssignL     | OpAssignKeywordExpr
                                                         | ExprCondL     action => ::first
 ExprAssignR   ::= ExprCond0   OpAssign  ExprAssignR     | ExprCondR     action => ::first
-ExprComma     ::= ExprAssignL OpComma   ExprComma       | ExprAssignR   action => ::first
+ExprComma     ::= ExprAssignL OpComma   ExprComma
+                | ExprAssignL OpComma   ExprComma OpComma | ExprAssignR   action => ::first
```

Comment:
This doesn't cover `[ $e, ]`, which I think may also be a valid thing to write, especially if `$e` is a long expression. How about this:
```
ExprComma     ::= ExprAssignL OpComma ExprComma | ExprAssignL OpComma | ExprAssignR
```

Reply:
Even better. I'll revise.

Reply:
Fixed.